### ISSUE TITLE
aria-expanded update is not announced by JAWS

### ISSUE THREAD

**Issue by gportela85** [FreedomScientific/VFO-standards-support#400]
I have a div with `aria-expanded="false"` that I toggle true/false on `enter` keydown.
And JAWS does not report this change in status.

> Brief description of the issue. Please list any specific steps.
> 
> Example:
> 1. Go to https://jsfiddle.net/8pjg1fao/
> 2. Click on `Group 1` and Tab to `Group 2` - the expanded status is announced.
> 3. Click Enter, the expand status changed but is not announced.
> 4. Shift-Tab and Tab back into `Group 2` - the new expanded status is announced.

### Expected result

> Expected the aria-expanded status change to be announced by JAWS

### Actual result

> Nothing happend

### Example

> https://jsfiddle.net/8pjg1fao/

#### JAWS version and build number
JAWS 2020.2006.12

 
#### Operating System and version
Windows 10

#### Browser and version:
Chrome 83
